test/ContentValidationListenerTest.php

```diff
@@ -129,7 +174,7 @@ public function testReturnsApiProblemResponseIfCollectionRequestWithoutBodyIsInv
         ]));
         $listener = new ContentValidationListener([
             'Foo' => ['GET' => 'FooValidator'],
-        ], $services);
+        ], $services, ['Foo' => 'foo_id']);
```

Comment:
Because without this it was entity request. Please check method `isCollection`:
https://github.com/zfcampus/zf-content-validation/blob/master/src/ContentValidationListener.php#L430-L432